specification/cognitiveservices/data-plane/FormRecognizer/preview/v1.0/FormRecognizer.json

```diff
@@ -0,0 +1,637 @@
+        "x-ms-examples": {
+          "Get Model from id": {
+            "$ref": "./examples/DeleteModelById.json"
+          }
+        }
+      }
+    },
+    "/custom/models/{id}/analyze": {
+      "post": {
+        "summary": "Analyze Form",
+        "description": "The document to analyze must be of a supported content type - 'application/pdf', 'image/jpeg' or 'image/png'. The response contains not just the extracted information of the analyzed form but also information about content that was not extracted along with a reason.",
+        "operationId": "AnalyzeWithCustomModel",
+        "consumes": [
+          "application/pdf",
+          "image/jpeg",
+          "image/png",
+          "multipart/form-data"
+        ],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "description": "Model Identifier to analyze the document with.",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          },
+          {
+            "name": "keys",
+            "in": "query",
+            "description": "An optional list of known keys to extract the values for.",
+            "required": false,
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "uniqueItems": false
+          },
+          {
+            "name": "form_stream",
+            "in": "formData",
+            "description": "A pdf document or image (jpg,png) file to analyze.",
+            "required": true,
+            "type": "file"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/AnalyzeResult"
+            }
+          },
+          "default": {
+            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Model from id": {
+            "$ref": "./examples/Analyze.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "TrainRequest": {
+      "description": "Contract to initiate a train request.",
+      "required": [
+        "source"
+      ],
+      "type": "object",
+      "properties": {
+        "source": {
+          "description": "Get or set source path.",
+          "maxLength": 2048,
+          "minLength": 0,
+          "type": "string"
+        }
+      }
+    },
+    "TrainResult": {
+      "description": "Response of the Train API call.",
+      "type": "object",
+      "properties": {
+        "modelId": {
+          "description": "Identifier of the model.",
+          "type": "string",
+          "format": "uuid",
+          "x-nullable": false
```

Comment:
This is the default. Please don't specify x-nullable: false.

Reply:
Noted. fixed.